templates/download/desktop/thank-you.html

```diff
@@ -442,24 +447,24 @@ <h3 class="p-card--title">Select topics you&rsquo;re interested in</h3>
 
   // Submit analytics before submitting form
   document.querySelector('#contributions-form').addEventListener(
-    'submit',
-    sendContributionFormAnalytics
+  'submit',
```

Comment:
Did you use prettier for this branch? There is some odd white space that has been removed.

Reply:
no, perhaps atom's auto indent lines